pkg/volume/cinder/cinder.go

```diff
@@ -163,6 +167,16 @@ func (plugin *cinderPlugin) newProvisionerInternal(options volume.VolumeOptions,
 	}, nil
 }
 
+func getCloudProvider(cloudProvider cloudprovider.Interface) (CinderProvider, error) {
```

Comment:
Provided cloudProvider implements CinderProvider, why do we care what specific type it is?